pymchelper/readers/topas.py

```diff
@@ -0,0 +1,272 @@
+from pathlib import Path
+import re
+from typing import List, Optional, Tuple
+
+import numpy as np
+from pymchelper.axis import MeshAxis
+from pymchelper.estimator import Estimator
+from pymchelper.page import Page
+from pymchelper.readers.common import Reader, ReaderFactory
+
+
+class TopasReaderFactory(ReaderFactory):
+    """Factory for TopasReader"""
+
+    def get_reader(self):
+        """Return TopasReader if the file extension is .csv"""
+        if self.filename.endswith('.csv'):
+            return TopasReader
+        return None
+
+
+def extract_parameter_filename(header_line: str) -> Optional[str]:
+    """Get parameter filename from the output file"""
+    parameter_filename = ""
```

Comment:
```suggestion
    parameter_filename = None
```

I think that returning None will more consistent with the `match.group` behaviour if not pattern is found

Reply:
Yes, I forgot to remove this line - it is overwritten with None anyway a few lines later